### PR TITLE
Update color variables (BDS) - 1

### PR DESCRIPTION
**Pull request by bscofield**

### CODE REVIEW

File: src/Pagehead.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const Pagehead = styled.div<SystemCommonProps & SxProp>`
   padding-top: ${get('space.4')};
   padding-bottom: ${get('space.4')};
   margin-bottom: ${get('space.4')};
-  border-bottom: 1px solid ${get('colors.border.gray')};
+  border-bottom: 1px solid ${get('colors.input.border')};
```
Comment (colebemis) [inline src/Pagehead.tsx:13]: I think `border.primary` might be a better fit here:

```suggestion
  border-bottom: 1px solid ${get('colors.border.primary')};
```

File: src/Caret.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import {get} from './constants'
```
Comment (colebemis) [inline src/Caret.tsx:3]: Why was this import added?

File: src/Popover.tsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ const PopoverContent = styled(BorderBox)`
   margin-right: auto;
   margin-left: auto;
   padding: ${get('space.4')};
-  background-color: ${get('colors.white')};
+  background-color: ${get('colors.bg.canvas')};
```
Comment (colebemis) [inline src/Popover.tsx:54]: Looks this should be `bg.overlay` ([reference](https://github.com/primer/css/blob/mkt/color-modes-whee/src/popover/popover.scss))
```suggestion
  background-color: ${get('colors.bg.overlay')};
```

File: src/Popover.tsx [hunk 4]
```diff
@@ -76,7 +76,7 @@ const PopoverContent = styled(BorderBox)`
     top: -14px;
     margin-left: -${get('space.2')};
     border: 7px solid transparent; // todo: solid
-    border-bottom-color: ${get('colors.white')};
+    border-bottom-color: ${get('colors.border.inverse')};
```
Comment (colebemis) [inline src/Popover.tsx:79]:
```suggestion
    border-bottom-color: ${get('colors.bg.overlay')};
```

File: src/Popover.tsx [hunk 5]
```diff
@@ -97,7 +97,7 @@ const PopoverContent = styled(BorderBox)`
     &::after {
       bottom: -14px;
       // stylelint-disable-next-line primer/borders
-      border-top-color: ${get('colors.white')};
+      border-top-color: ${get('colors.border.inverse')};
```
Comment (colebemis) [inline src/Popover.tsx:100]:
```suggestion
      border-top-color: ${get('colors.bg.overlay')};
```

File: src/Popover.tsx [hunk 6]
```diff
@@ -176,7 +176,7 @@ const PopoverContent = styled(BorderBox)`
     &::after {
       right: -14px;
       // stylelint-disable-next-line primer/borders
-      border-left-color: ${get('colors.white')};
+      border-left-color: ${get('colors.border.inverse')};
```
Comment (colebemis) [inline src/Popover.tsx:179]:
```suggestion
      border-left-color: ${get('colors.bg.overlay')};
```

File: src/Popover.tsx [hunk 7]
```diff
@@ -192,7 +192,7 @@ const PopoverContent = styled(BorderBox)`
     &::after {
       left: -14px;
       // stylelint-disable-next-line primer/borders
-      border-right-color: ${get('colors.white')};
+      border-right-color: ${get('colors.border.inverse')};
```
Comment (colebemis) [inline src/Popover.tsx:195]:
```suggestion
      border-right-color: ${get('colors.bg.overlay')};
```

File: src/Popover.tsx (hunk 3 above)
Comment (colebemis) [reply src/Popover.tsx:54]: Also, usages of `get('popovers.colors.caret')` in this file should be replaces with `get('colors.border.overlay')`

File: src/Caret.tsx (hunk 2 above)
Comment (bscofield) [reply src/Caret.tsx:3]: ack, leftover from an early exploration

**Comment by bscofield** [comment]
https://github.com/github/design-systems/issues/1219 (x-ref)

**Review by colebemis — APPROVED** [review]
🎉
